components/automate-chef-io/content/docs/monitoring.md

```diff
@@ -0,0 +1,56 @@
++++
+title = "Monitoring Chef Automate"
```

Comment:
I didn't add a corresponding entry to the `config.toml` in `components/automate-chef-io`; let me know if I should.

Reply:
You don't necessarily need a corresponding entry. I did, however, move this entry to be under the Reference section.